src/lib/utils.js

```diff
@@ -488,6 +488,18 @@ const validateActorName = (actorName) => {
     }
 };
 
+/**
+ * Detects language of actor based on the files present in its directory
+ * @param {string} actorDirectory Path to the actor's root directory
+ * @return {string} The language of the actor, or 'uknown' for unknown language
+ */
+const detectActorLanguage = (actorDirectory) => {
```

Comment:
We were talking about having a `scripts` section in `actor.json`, where you could specify the lifecycle scripts, and then you'd have the `post-create` and `run` scripts specified explicitly. But we first have to agree on the specification for that, so until then, this will do.